examples/ocmd/ocmd-server-1.toml

```diff
@@ -77,21 +77,18 @@ providers = "providers.demo.json"
 driver = "memory"
 
 [grpc.services.appprovider]
-driver = "demo"
-iopsecret = "testsecret"
-wopiurl = "http://0.0.0.0:8880/"
-wopibridgeurl = "http://localhost:8000/wopib"
+driver = "wopi"
+
+[grpc.services.appprovider.drivers.wopi]
+iop_secret = "hello"
+wopi_url = "http://0.0.0.0:8880/"
+wopi_bridge_url = "http://localhost:8000/wopib"
```

Comment:
`wopi_bridge_url` is gone now!
Shall we add `ssl_verify = "True"`, cf. https://github.com/cs3org/reva/pull/1842 ?